### PR TITLE
BXMSDOC-3457 (Process Administration, Githook notifications & SSH Keys)

### PR DESCRIPTION
**Pull request by gadeyredhat**
[Rendered Output](http://file.pnq.redhat.com/~gadey/BXMSDOC-3457/)

[Jira Id](https://issues.jboss.org/browse/BXMSDOC-3457)

### CODE REVIEW

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+You can customize the default pagination option in {CENTRAL} by editing the _Default items per page_ property on the *Process Administration* page.
+
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *Process Administration*.
+. From the *Properties* section, update the _Default items per page_ property and click *Save*.
++
+[NOTE]
+====
+The allowed values for displaying the default items per page are 5, 10, 15, 20, 50 or 100.
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: This needs to be fixed in the help text as well but allowed values are: 10, 20, 50 and 100.

**Review by pefernan — APPROVED** [review]
+1 for ssh & git hooks notification

**Comment by emmurphy1** [comment]
Don't use per, use 'for each'. Comma after 50:
...50, or 100.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
+[id='managing-business-central-customizing-githook-notifications-proc']
+= Customizing git hook notifications
+
+{CENTRAL} provides a mechanism to enable users to get feedback about git hook executions using customized
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: to enable, change to 'that enables' 
change get to receive
I don't think exit code needs to be in code example text.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+messages based on a hook's `exit code`.
+
+.Procedure
+
+. To enable notifications, in the `standalone.xml` file, set the `appformer.git.hooks.bundle` system property to point to a `*.properties` file containing the custom messages for each `exit code`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: remove comma after file:
To enable notifications, in the `standalone.xml` file  set the ...

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
++
+For example,
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: For example:

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+[source]
+----
+<system-properties>
+  <property name="appformer.git.hooks.bundle" value="/opt/jboss-as/git-hooks-messages/Messages.properties">
+  </property>
+  ...
+</system-properties>
+----
+
+. For displaying the messages, each line of the `*.properties` should have the following format:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: To display the messages...
should have or must have? 

Reorganize:

<exit code>=<display message>

In the preceding example, `<exit code>` is the git hook exit code and <display message> is the custom message that is displayed to a user.

Because these are variables I would use under scores:
exit_code, display_message.

Also, note will be = is

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
++
+`<exit code>=<display message>`
++
+where
++
+`<exit code>` is the git hook exit code
++
+`<display message>` is the custom message that will be displayed to a user.
++
+For example:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: This isn't an example of how to format the .properties file, but an example of out. So either provide an example for the code or say something like:
Exit code messages with this format appear similar to the following example:

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc [hunk 7]
```diff
@@ -0,0 +1,20 @@
+[id='managing-business-central-githook-notifications-exit-codes-types-con']
+= Types of exit codes
+
+{CENTRAL} has the following three types of exit codes:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc:4]: Can you tell me what an exit code is? And I wonder if this concept module should precede the procedure. Shouldn't I know about exit codes before I customize them?

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+.Successful notification
+`0`: Success. A git hook that exits with code `0` will display a success notification on the user interface.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc]: will display = displays
in the UI

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+.Warning notification
+`1..30`: Warning. A git hook that exits with a code between `1` and `30` (included) will display a warning notification on user interface.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc]: (included) = (inclusive)

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+.Error notification
+Anything else: Error. Any other code number will be displayed as an error notification.
+
+[IMPORTANT]
+====
+*UNIX* machines only support error codes between 0 (success) to 255 (error), any exit code outside of this range will
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc]: Why is UNIX bolded?
machines = systems
between 0 (success) and 255 (error)
New sentence:
Any exit code...
..outside of this range is converted to which might (may is for permission)
Windows, why bold... systems as above

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc [hunk 11]
```diff
@@ -0,0 +1,7 @@
+[id='managing-business-central-githook-notifications-internationalize-con']
+= Internationalize git hook notifications
+
+You can also internationailze the notification messages by placing different `*.properties` files in the directory specified for the `appformer.git.hooks.bundle` system property. The names of the localized files should be `<filename>_<lang>.properties`, and it should contain the same entries as defined in the the original one.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc]: remove also
The names of the ...files have the format... These files (should or must) contain the same...as in the original file.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+
+For example, you have specified the system property to point to `Messages.properties`. Now you can create localized message properties files like `Messages_en.properties` for English, `Messages_fr.properties` for French, `Messages_it.properties` for Italian, and so on. The notification service will choose the one based on the user language and if there are no available translations
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc:6]: I think this would work better as a procedure.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc [hunk 13]
```diff
@@ -0,0 +1,4 @@
+[id='managing-business-central-process-administration-con']
+= Process administration
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc:2]: Title is a little vague. Maybe {CENTRAL} process administration

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc [hunk 14]
```diff
@@ -0,0 +1,4 @@
...
+
+Users can define the number of items to be displayed by default for all users under *Manage* (process definitions, process instances, tasks, jobs, and execution errors) and *Task Inbox* sections. You can set the default pagination using *Process Administration* option from the *Admin* settings page in {CENTRAL} and once a new default is applied it is effective for all users who do not have a custom preference saved for these items.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc]: in the *Manage*.
I would start this para with 'You' as you use it in the second sentence. So.. You can define....
using the *Pagination...
New sentence:
After a new default is applied, it..

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc [hunk 15]
```diff
@@ -0,0 +1,14 @@
+[id='managing-business-central-process-administration-proc']
+= Customizing process administration
+
+You can customize the default pagination option in {CENTRAL} by editing the _Default items per page_ property on the *Process Administration* page.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: If Default items per page is a label in the UI, it should be bold.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *Process Administration*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: I would say Select instead of go to

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+. From the *Properties* section, update the _Default items per page_ property and click *Save*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: As above, Default items...

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc (hunk 1 above)
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: Maybe rephrase (and search for other instances):

You can specify 5, 10, 15, 20, 50, or 100 items to display on each page.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc [hunk 18]
```diff
@@ -0,0 +1,6 @@
+[id='managing-business-central-ssh-keys-con']
+= SSH Keys
+
+{Central} provides a SSH keystore service for enabling SSH authentication for users. It provides a configurable default SSH keystore, extensible APIs (for custom implementations), and support for multiple SSH public keys formats.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: ...service to enable user SSH authentication.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 19]
```diff
@@ -0,0 +1,44 @@
+[id='managing-business-central-ssh-keys-create-proc']
+= Creating SSH keys
+
+Before you can add or register SSH keys to {CENTRAL}, you need to generate one on your machine using the command terminal.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: need to = must
must generate an SSH key on your system. (no need to specify using the terminal)

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 20]
```diff
@@ -0,0 +1,44 @@
...
+
+.Procedure
+. Open a command terminal on your machine.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: I think just 'terminal'. Lose command.
machine = system

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+. Run the `ssh-keygen` command to create the key.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Run the `ssh-keygen` command to create the SSH key as shown in the following example, where <user_login> is your (well, I am unsure. Do I put my username and password, or just my password)?

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
++
+[source]
+----
+ssh-keygen -t rsa -b 4096 -C "<your_user_login_here>"
+----
++
+[NOTE]
+====
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc:16]: use code format for `ssh-dss`, `ecda-sha...` etc.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 23]
```diff
@@ -0,0 +1,44 @@
...
+The SSH key formats supported by {CENTRAL} keystore are ssh-rsa, ssh-dss, ecdsa-sha2-nistp256, ecdsa-sha2-nistp384, and ecdsa-sha2-nistp521.
+====
+. When prompted, press enter and accept the default key file location.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Is this the Enter key? If so, press the Enter key, or if its' on the UI: Click *Enter*

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 24]
```diff
@@ -0,0 +1,44 @@
...
++
+[source]
+----
+Enter a file in which to save the key (/home/<your_login_here>/.ssh/id_rsa): [Press enter]
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: So is <your_login_here> part of the UI? Is so, please use as you have in the preceding example.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 25]
```diff
@@ -0,0 +1,44 @@
...
+----
+. At the command prompt, enter and confirm the pass phrase.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: passphrase

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc [hunk 26]
```diff
@@ -0,0 +1,44 @@
...
+Enter same passphrase again: [Type passphrase again]
+----
+. Start the `ssh-agent`.
++
+[source]
+----
+eval "$(ssh-agent -s)"
+Agent pid <any-number-here>
+----
+. Add the new SSH private key to the ssh-agent. If you have used a different key name, replace id_rsa with your key name.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: `ssh-agent`
If you used a different... 
id_rsa in code

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc [hunk 27]
```diff
@@ -0,0 +1,11 @@
+[id='managing-business-central-ssh-keys-delete-proc']
+= Deleting a SSH key
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc]: Deleting an SSH key

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc [hunk 28]
```diff
@@ -0,0 +1,11 @@
...
+
+You can delete a SSH key from {CENTRAL} by from the *SSH Keys* page.
+
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *SSH Keys*.
+. All the SSH keys are listed on the *SSH Keys* page.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc]: All of the

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc [hunk 29]
```diff
@@ -0,0 +1,11 @@
...
+. To delete a SSH key, click on the delete icon.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc]: an SSH (search doc)
click not click on
Maybe give me another hint about the delete icon. For example, ...the delete icon (trash can).

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc [hunk 30]
```diff
@@ -0,0 +1,25 @@
+[id='managing-business-central-ssh-keys-register-proc']
+= Registering your SSH public key with the SSH keystore
+
+You must register your newly created SSH public key with {CENTRAL}'s SSH keystore.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc]: Usually we try to use possessive's with objects. So I would write:
with the {CENTRAL} keystore.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc [hunk 31]
```diff
@@ -0,0 +1,25 @@
...
+
+.Prerequisite
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc]: This is not a prereq, it is a procedure in it's own right. I would incorporate these steps with the actual procedure. In anycase, you  need to remove them from the prereq section.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc [hunk 32]
```diff
@@ -0,0 +1,25 @@
...
+
+* On a command terminal, use the cat command to display your key content. Copy the contents of your SSH public key. If you have used a different key name, replace `id_rsa` with your key name, and copy it.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc]: In a terminal.
`cat`
to display the contents of your SSH public key.
have used = used
Not clear what I should copy. The name?

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc [hunk 33]
```diff
@@ -0,0 +1,25 @@
...
++
+[source]
+----
+cat ~/.ssh/id_rsa.pub
+----
+
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *SSH Keys*.
+. From the *SSH Keys* page, click *Add SSH Keys*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc:17]: On the *Admin...

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc [hunk 34]
```diff
@@ -0,0 +1,25 @@
...
+. In the *Add SSH Key* window, enter the *Name* and copy the contents of the SSH public key to the *Keys* field.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc]: ...enter a name in the *Name* field..
Although I prefer to use box instead of field.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc [hunk 35]
```diff
@@ -0,0 +1,25 @@
...
++
+[NOTE]
+====
+Both *Name* and *Key* are mandatory fields.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc]: The *Name* and the *Key* fields are  mandatory.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc [hunk 36]
```diff
@@ -0,0 +1,9 @@
+[id='managing-business-central-ssh-keystore-custom-con']
+= Custom SSH keystore
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc:2]: Customizing the SSH keystore?

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc [hunk 37]
```diff
@@ -0,0 +1,9 @@
...
+
+You can extend and customize the default SSH keystore as per your specific requirements. Use the `appformer.ssh.keystore` system property to specify the Java class name of the SSH service to use. If this property is not defined or it contains a wrong value then the default SSH keystore is loaded.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc]: as per your = according to your requirements. No need for 'specific'.
...or it contains an incorrect value...

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc [hunk 38]
```diff
@@ -0,0 +1,30 @@
+[id='managing-business-central-ssh-keystore-default-con']
+= Default SSH keystore
+
+The default SSH keystore included with {CENTRAL} provides a file-based storage mechanism to store a user's public keys. By default, {CENTRAL} uses the `*.security` folder as the root directory. However, it is possible to use a custom storage path by setting the value of the `appformer.ssh.keys.storage.folder` system property point to a different folder.
+
+The SSH public keys are stored in the `{securityFolderPath}/pkeys/{userName}/` folder structure.
+
+Each SSH public key consists of a pair of following files in the storage folder:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc]: I was a little confused by 'a pair of'. Do you mean:
Each SSH public consists of the following files, located in the storage folder: ?

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc [hunk 39]
```diff
@@ -0,0 +1,30 @@
...
+
+* `{keyId}.pub`: This file contains the SSH public key content. As the file name determines the logic key ID on the system, ensure that the file name is not modified during runtime.
++
+For example,
```
Comment (emmurphy1) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc]: For example:

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc (hunk 1 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: Changed as suggested.

Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: Rephrased as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc (hunk 2 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc (hunk 3 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc (hunk 4 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc (hunk 5 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc (hunk 6 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc (hunk 35 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc (hunk 36 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc:2]: @emmurphy1, not changed.
Reason:
- What I mean to say here is that there are two types of SSH keystore, default keystore & custom keystore.
- Another reason I am not using the "Customizing the SSH keystore" name is that this is a concept module.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc (hunk 37 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-custom-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc (hunk 38 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc (hunk 39 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc (hunk 8 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc (hunk 10 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc (hunk 11 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc (hunk 12 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-internationalize-con.adoc:6]: Imo, this is better suited as a narrative rather than a procedure.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc (hunk 7 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc:4]: @emmurphy1, Changed as suggested. I created a separate "exit code" definition concept module and moved the exit code types concept module to before the procedure.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc (hunk 13 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc:2]: Changed as suggested. Just fyi, "Process Administration" is the name of the option in the BC Admin settings.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc (hunk 14 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc (hunk 15 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc (hunk 16 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc (hunk 17 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc (hunk 18 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 19 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 20 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 21 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 22 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc:16]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 23 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 24 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: @emmurphy1, Changed as suggested.
Just fyi <your_login_here> gets replaced by your username here. for e.g. (/home/gadey/.ssh/id_rsa).

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 25 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc (hunk 26 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-create-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc (hunk 27 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc (hunk 28 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc (hunk 29 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-delete-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc (hunk 34 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-register-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc [hunk 40]
```diff
@@ -0,0 +1,6 @@
+[id='managing-business-central-ssh-keys-con']
+= SSH Keys
+
+{Central} provides an SSH keystore service to enable user SSH authentication. It provides a configurable default SSH keystore, extensible APIs (for custom implementations), and support for multiple SSH public keys formats.
+
+You can access the *SSH Keys* option from the *Admin* page to enable users to register their SSH public keys.
```
Comment (domhanak) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: I think that admin is not enabling anything @pefernan  is he?

**Review by domhanak — COMMENTED** [review]
Looks good, just one comment

File: doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc (hunk 40 above)
Comment (pefernan) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: @domhanak I think that pars refers to the "admin page" so the bold shoud be *Admin page*. @gadeyredhat WDYT??

Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: @domhanak, yes, admin refers to the **Admin** page here. 
@pefernan, as per writing style guide only the name (a noun) of UI should be in bold and not the UI element. For example, say a screen has three tabs, Roles, Groups, & Users. Then only **Roles**, **Groups**, & **Users** should be in bold but not tab.

Comment (domhanak) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: > You can access the *SSH Keys* option from the *Admin* page to enable users to register their SSH public keys. 

The issue here is that we are saying: `you` (who? Admin?) can access `SSH Keys` option to enable `users` to register their SSH public keys. Imho this does not make sense because there is no option that enables SSH Keys usage for other users.

Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: @domhanak, ok I guess I undestand now what you are trying to say. Then can I change it to the following:
_You can access the SSH Keys option from the **Admin** page to register your SSH public keys._
Is the sentence now clear in your opinion?

Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc]: Changed as suggested.
